### PR TITLE
update to 4.3.0

### PR DESCRIPTION
**Pull request by mcg1969**
- updated version and sha
- added skip for python < 3.7, removed python bounds

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1a9462dcc3347a79b1f1c0271fbe79e844580bb598bafa1ed208b94da3cdcd42
+  sha256: {{ sha256 }}
 
 build:
   number: 0
+  skip: true  # [py<37]
```
Comment (mcg1969) [inline recipe/meta.yaml:15]: Python 3.7 requirement found here:
https://github.com/python/typing_extensions/blob/main/pyproject.toml#L12

File: recipe/meta.yaml [hunk 2]
```diff
@@ -20,12 +22,12 @@ outputs:
```
Comment (cbouss) [inline recipe/meta.yaml:22]: Please remove noarch from both typing_extensions and typing-extensions.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -20,12 +22,12 @@ outputs:
...
+        - python
         - pip
         - setuptools
         - wheel
       run:
-        - python >=3.6
+        - python
 
     test:
       imports:
```
Comment (cbouss) [inline recipe/meta.yaml:33]: home should be https://github.com/python/typing_extensions
about/doc_url should be https://typing.readthedocs.io

**Review by cbouss — CHANGES_REQUESTED** [review]
Please remove noarch, otherwise LGTM.

File: recipe/meta.yaml (hunk 2 above)
Comment (mcg1969) [reply recipe/meta.yaml:22]: oh yes, thanks!

File: recipe/meta.yaml (hunk 3 above)
Comment (mcg1969) [reply recipe/meta.yaml:33]: excellent!

Comment (mcg1969) [reply recipe/meta.yaml:33]: @cbouss I don't agree with the doc_url. That's for the standard Python typing information, not for this typing_extensions package in particular. I've changed it to https://github.com/python/typing_extensions/README.md

**Comment by mcg1969** [comment]
@cbouss fixes made, though with one modification explained above! thank you so much!

File: recipe/meta.yaml (hunk 3 above)
Comment (cbouss) [reply recipe/meta.yaml:33]: My bad, I followed blindly pyproject.toml. The link you put is 404 though. It should be this: https://github.com/python/typing_extensions/blob/main/README.md

**Review by cbouss — COMMENTED** [review]
LGTM, if the doc_url link is corrected to https://github.com/python/typing_extensions/blob/main/README.md

File: recipe/meta.yaml (hunk 3 above)
Comment (mcg1969) [reply recipe/meta.yaml:33]: got it

**Comment by mcg1969** [comment]
fixing the link. thaks again @cbouss

**Review by cbouss — APPROVED** [review]
LGTM

**Pull request merged**